Dockerfile

```diff
@@ -31,4 +31,4 @@ COPY \
   /opt/broker/plugins/lib/realmq_adapter
 
 CMD ["/run.sh"]
-EXPOSE 1883 44053 8888 8080
```

Comment:
**question**: @baranga Don't we also need to export the websocket port `8080`?

Reply:
We do not necessarily need to expose it via Dockerfile but we should. I'll adjust that.

Reply:
:heavy_check_mark: Done